### ISSUE TITLE
Dropdown does not propagate aria-labelledby in option list

### ISSUE THREAD

**Issue by NatPri** [OfficeDev/office-ui-fabric-react#8878]
## Codepen demonstrating the issue
https://codepen.io/natpri/pen/xeezyo

## Actual behavior:
The dropdown callout containing the options list with the role of listbox does not have a label associated with it. This appears to be due to using the wrong spelling in [Dropdown.base.tsx](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx#L493) (it should be "ariaLabelledBy" for FocusZone to pick it up correctly).

## Expected behavior:
The dropdown callout should have an `aria-labelledby` attribute, since it has a `label`. Alternately, if the user doesn't specify a label on the dropdown, but does specify an `ariaLabel` property, it should propagate that ariaLabel to the dropdown callout, or there should be a mechanism to set that label.

## Documentation describing expected behavior
https://www.w3.org/TR/wai-aria-1.1/#listbox
https://www.w3.org/TR/wai-aria-practices-1.1/#Listbox

**Comment by atneik** [comment]
@NatPri apologies for the delayed response. I earlier thought I understood the issue fully and will be able to fix it, but upon further investigation its unclear to me. 

I see the component behaving as per your expectation. In your example codepen, the `ariaLabel` is showing up correctly in the DOM:

![image](https://user-images.githubusercontent.com/785361/57261594-35c55c80-701d-11e9-83cb-6c07c386b04e.png)

**Comment by NatPri** [comment]
The issue I'm seeing is that when the dropdown is activated, the callout exposes an element with a `role="listbox"` that doesn't have a label, as seen in this DOM snippet. The code I linked to tries to set an `aria-labelledby` property on it, but fails to do so.

![image](https://user-images.githubusercontent.com/25355689/57327286-69f35880-70c3-11e9-88fb-b255f56efa00.png)

**Comment by NatPri** [comment]
(also apologies if having the codepen with both normal label and ariaLabel confused the issue)
